tests/dbos-runtime/runtime.test.ts

```diff
@@ -150,6 +150,18 @@ describe("runtime-tests", () => {
     await waitForMessageTest(command, "1234");
   });
 
+  test("runtime hello with appDir provided as CLI parameter", async () => {
+    process.chdir("../../../..");
+    try {
+      const command = spawn("dist/src/dbos-runtime/cli.js", ["start", "--appDir", "packages/create/templates/hello"], {
```

Comment:
Does it mean we're able to use a global installation of the `dbos-sdk` to run apps everywhere? What if the DBOS version is different from the one the app is compiled with?

Reply:
Yes, and it will run with the global installation, whatever version that is.

Reply:
I think it might cause undesired behaviors if the versions don't match. Maybe we can print out a warning if versions don't match?

Reply:
That's not easy to check reliably